src/core/config/indices/od_context.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+#include "config/common_option.h"
+#include "config/indices/type.h"
+
+namespace config {
+extern CommonOption<IndicesType> const kODContextOpt;
```

Comment:
'ODContext' option is useful only for OD verifier and won't be needed by other algorithms. So I think that it should be removed. Instead, you can use `Option` as shown [here](https://github.com/Desbordante/desbordante-core/blob/main/src/core/algorithms/ind/faida/faida.cpp#L18).